mkdocs/docs/configuration.md

```diff
@@ -228,19 +228,19 @@ catalog:
 catalog:
   default:
     uri: thrift://localhost:9083
-    s3.endpoint: http://localhost:9000
-    s3.access-key-id: admin
-    s3.secret-access-key: password
+    hive:
```

Comment:
Have you tested it with this additional layer? I know that it is equivalent in yaml, but just want to double check

Reply:
Sorry, at first I test only against environment variable, but the latest change I test on both `env` and the `.pyiceberg.yaml` file